src/DeviceModeling/device_modeler.h

```diff
@@ -1212,9 +1212,9 @@ class device_modeler {
       if (drv)
         net_ptr->set_source(drv);
       else
-        std::cout << "WANRN: The driver " << driver_name << " for net "
-                  << net_name << " does not exis in block " << block_name
-                  << std::endl;
+        std::cout << "WANRN: The driver \"" << driver_name << "\" for net \""
```

Comment:
@manadher , please correct when you get a chance the spelling mistakes:

WANRN -> WARN

exis -> exist